fhd_core/source_modeling/source_dft_multi.pro

```diff
@@ -129,14 +129,19 @@ IF Keyword_Set(n_spectral) THEN BEGIN
     
     IF Keyword_Set(dft_threshold) THEN BEGIN
         model_uv_arr=Ptrarr(n_pol,n_spectral+1)
-        edge_i = where((x_vec LT dft_edge_pix) OR (y_vec LT dft_edge_pix) OR (dimension-x_vec LT dft_edge_pix) OR (elements-y_vec LT dft_edge_pix),$
-            n_edge_pix, complement=center_pix_i, ncomplement=n_center_pix)
+        if ~keyword_set(gaussian_source_models) then begin
+            edge_i = where((x_vec LT dft_edge_pix) OR (y_vec LT dft_edge_pix) OR (dimension-x_vec LT dft_edge_pix) OR (elements-y_vec LT dft_edge_pix),$
+                n_edge_pix, complement=center_pix_i, ncomplement=n_center_pix)
+        endif else begin
+            edge_i = where((x_vec LT dft_edge_pix) OR (y_vec LT dft_edge_pix) OR (dimension-x_vec LT dft_edge_pix) OR (elements-y_vec LT dft_edge_pix) OR $
+                gaussian_x,n_edge_pix, complement=center_pix_i, ncomplement=n_center_pix)
```

Comment:
Won't this flag everything except a column of pixels along gaussian_x=0?

I think the correct way to do this is to make a `dft_threshold_use` variable (maybe with a better name) that is forced to 0 if `gaussian_source_models` is set.

Reply:
Sorry for the late reply.

gaussian_x is a vector across all sources, with each element describing the width of the gaussian source sigma in pixels in the x-direction. If the a particular source is a point source, then this is set to 0 for that element. That way, we can easily pick out which sources are gaussians, and filter them into source_dft instead of fast_dft. 

tested and worked.